client/account-settings/components/label-payment-method/style.scss

```diff
@@ -0,0 +1,38 @@
+.card.is-compact.label-payment-method {
```

Comment:
Why do we need both `.card` and `.is-compact` here? Previously we were only using `.card`.